js/main.js

```diff
@@ -826,14 +826,22 @@ function createAttributeBalanceEntry(balanceElement, getEffectFn, getEffectsFn,
 
     const balanceEntryElement = Dom.new.fromTemplate('balanceEntryTemplate');
     const domGetter = Dom.get(balanceEntryElement);
-    domGetter.byClass('name').textContent = '(' + name + ')';
+    const descriptionElement = domGetter.byClass('name');
+    const updateDescription = () => {
+        const description = typeof name === 'function' ? name() : name;
```

Comment:
```suggestion
        const description = isFunction(name) ? name() : name;
```
Haben so ein paar type helper in utils.js